src/core/useTrackBindingPlugin.js

```diff
@@ -62,7 +63,8 @@ export class TrackBindingPlugin {
             return;
         }
 
-        let elem = event.target || event.srcElement;
+        var originalElem = event.target || event.srcElement;
```

Comment:
Do you need this new variable? 
Just pass `elem` Into `callback`
Also, prefer let to var